apps/stable_diffusion/scripts/txt2img.py

```diff
@@ -224,6 +224,7 @@ def txt2img_inf(
             args.use_base_vae,
             args.use_tuned,
         )
+        txt2img_obj.enable_attention_slicing("auto")
```

Comment:
can we please add it as an arg.attention_slicing and set it to auto / max ? You will also need to add it to the web/index.py.